src/main/java/com/google/alpollo/model/SongEntity.java

```diff
@@ -62,10 +68,25 @@ public boolean equals(Object obj) {
   @Override
   public int hashCode() {
     int hash = 7;
-    hash = 31 * hash + (int) salience;
     hash = 31 * hash + (name == null ? 0 : Objects.hashCode(name));
     hash = 31 * hash + (type == null ? 0 : Objects.hashCode(type));
     hash = 31 * hash + (wikiLink == null ? 0 : Objects.hashCode(wikiLink));
     return hash;
   }
+
+  public String getName() {
+    return name;
+  }
+
+  public Set<String> getType() {
+    return type;
+  }
+
+  public double getSalience() {
+    return salience;
+  }
+
+  public void setSalience(double newSalience) {
+    this.salience = newSalience;
```

Comment:
nit/optional: when the parameter (i.e. `newSalience`) name is different than the field (i.e. `salience`) you can omit `this`.  So here you could have `salience = newSalience`. If the parameter was also called `salience`, you'd need `this` to differentiate them, i.e. `this.salience = salience;`.